src/main/java/com/robin/hoteless/model/Reservation.java

```diff
@@ -0,0 +1,119 @@
+package com.robin.hoteless.model;
+
+import java.time.LocalDate;
+import java.util.Date;
+
+public class Reservation {
+    private int id;
+    private Date reservationCreationDate;
+    private int lengthOfStay;
+    private double dailyRate;
+    private LocalDate checkInDate;
+    private LocalDate checkOutDate;
+    private String roomType;
+    private double totalCostOfStay;
```

Comment:
Including taxes, fees, 2oz alcohol bottles out of the mini-fridge, damage costs for that crowbar through the bathtub, the 5 lost keys, and the baby poop that dripped onto the carpet? ;) Perhaps not the _total_ cost of the stay...